### ISSUE TITLE
can I use TieDIE applied to analyze different level of proteome, such as proteome and phosphoproteome 

### ISSUE THREAD

**Issue by pariswu1988** [epaull/TieDIE#5]

**Comment by epaull** [comment]
Yes, either or both will work! Decide how many separate input sets you want--i.e. RPPA protein quantity could be input set 1, phosphoproteome could be set 2, and you could combine those all with other data. Each set will diffuse over the network separately and then be combined using the TieDIE algorithm. Any other specific questions?

**Comment by zhenzuo2** [comment]
@epaull Thanks! Could you please elaborate on how to provide separate input sets to TieDIE? Should I run TieDEI on each set seperatly and combine the network together in the end?

**Comment by epaull** [comment]
Each separate file is a different input set, and you can do more than 2
inputs if you want—just make sure a different file for each! I think the
old version of the tiedie (master branch maybe) takes two inputs —up_heats
and —down_heats, but the newer one can take all input files (2 or more) as
command line arguments

Does that help?

Evan

**Comment by zhenzuo2** [comment]
That helps a lot, Evan. Got it to work with python 3.7 on a Linux machine.
